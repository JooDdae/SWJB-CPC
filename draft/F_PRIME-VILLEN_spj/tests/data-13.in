139955722875827748775495381332997816521
