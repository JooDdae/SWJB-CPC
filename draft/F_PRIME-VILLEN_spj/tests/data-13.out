16
222 3 3 4 340 3 144 4 93 958 709 2 2 10 1206 843 